### DISCUSSION TITLE
What does imbox error 255 mean?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#3230] [Q&A Feedback] [answered]
The pre-release of 1.11.0 contains a new feature. I wanted to test the feature, but whenever I create an encrypted box, it ends with "**imbox error 255**".

**Comment by DavidXanatos** [comment]
does this happen from the new box wizard?
do you have the imdisk driver installed?

**Reply by bastik-1001** [reply]
Yes, with the new box wizard.

I reinstalled the addon again, but I am getting:
`SBIE2245 Failed to mount box image, ImBox error 255`

I am able to use the tool (RamDiskUI) to create a Ram disk, which shows up with a drive letter.

**Reply by DavidXanatos** [reply]
that is strange, can that error should on that case only occur when an invalid argument is passed to the ImBox.exe
could you use taskexplorer or systeminformer to capture the commandline sbiesvc.exe passes to imbox.ese and post it

**Reply by bastik-1001** [reply]
I tried twice, but there does not appear to be something for me to observe. I made a screenshot.

![Taskexplorer](https://github.com/sandboxie-plus/Sandboxie/assets/1449429/a85a8dd6-a83d-46cd-aeea-0037737ad588)

**Reply by DavidXanatos** [reply]
seams on your system imbox is crashing not sure why could you look for a crash dump?

**Reply by bastik-1001** [reply]
It does not appear to create these dumps.

I created a registry key for the executable, so it would create one, but it does not.

Well never mind, I just wonder if someone else is seeing this. If that is just me, I won't say that this is an issue.

**Reply by DavidXanatos** [reply]
it is surely an issue we just need to find out why, we can do a team viewer session and use a debug build to test it in detail, 
do you have visual studio installed?

**Reply by bastik-1001** [reply]
I have not enough time to do that right now. And I don't have Visual Studio installed. The system is still a Windows 7 machine.

If that is not something that effects other users, it is probably not worth the time.

**Reply by DavidXanatos** [reply]
I'll test it on a windows 7 machine then...

**Reply by bastik-1001** [reply]
Now, I can at least provide a crash dump: [https://github.com/bastik-1001/Crashdumps/blob/main/ImBox.exe.6620.dmp](https://github.com/bastik-1001/Crashdumps/blob/main/ImBox.exe.6620.dmp)

**Reply by isaak654** [reply]
@bastik-1001 Please download the new test build from [here](https://github.com/sandboxie-plus/Sandboxie/actions/runs/5978646060) and report back.

**Reply by bastik-1001** [reply]
With the linked test build it still fails. I have another dump [https://github.com/bastik-1001/Crashdumps/blob/11a/ImBox.exe.8960.dmp](https://github.com/bastik-1001/Crashdumps/blob/11a/ImBox.exe.8960.dmp)

**Reply by isaak654** [reply]
It seems that it might be related to a conflict with 0patch (0patchLoaderX64.dll), which is present in both dumps.
If so, David should be able to reproduce it.

**Reply by bastik-1001** [reply]
With the first pre-release, I had disabled 0patch as I assumed it could be an issue. Now I disabled 0patch again.

Those are the dumps with it being disabled. [https://github.com/bastik-1001/Crashdumps/blob/0patch/ImBox.exe.7636.dmp](https://github.com/bastik-1001/Crashdumps/blob/0patch/ImBox.exe.7636.dmp) and [https://github.com/bastik-1001/Crashdumps/blob/0patch/ImBox.exe.6704.dmp](https://github.com/bastik-1001/Crashdumps/blob/0patch/ImBox.exe.6704.dmp

**Reply by bastik-1001** [reply]
Does someone else have this issue? If that is just my quirky system, I won't want anyone to spend time on it.

**Reply by bastik-1001** [reply]
I uninstalled the software (0patch) and tried again. [https://github.com/bastik-1001/Crashdumps/blob/nopatch/ImBox.exe.1096.dmp](https://github.com/bastik-1001/Crashdumps/blob/nopatch/ImBox.exe.1096.dmp) I notice that the dump is smaller.

Edit: I will not be re-installing 0patch, soon. This will be something for later.

**Reply by DavidXanatos** [reply, marked as answer]
I got my win 7 test vm up to date and tested it, it seams that there is a general windows 7 incompatibility, will look into it, its not related to 0 patch, the VM ist clean only visual studio and the latest MSFT updates

**Reply by DavidXanatos** [reply]
fixed on build 1.11.0d

**Reply by bastik-1001** [reply]
Good to hear that this is fixed, this should be perceived well, by those running Windows 7 and that will spend extra money on the feature. :)

Once there is another (pre-)release I might try again. Thank you for putting effort in to keep the Windows 7 compatibility intact.
